frontend/src/components/TextField.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components/macro'
+import Label from './Label'
+export default function TextField({
+  type = 'String',
```

Comment:
@lindatroesken I wonder what input type String does :), you can leave it blank or set it to text, which is the default

Reply:
Thanks :). I guess I was thinking in Java instead of JavaScript... I will change that